### PR TITLE
Misc files update for upcoming UPU release

### PR DESCRIPTION
**Pull request by NovaRain**
* Update sfall (again) to the latest 4.4.0.1 for fixing XP/2003 compatibility.
* Remove non-existing global scripts from manual install batch/shell script.
* For some reason there was a LF near the end of `credits.txt` (not present in vanilla file).
* `FadeBackgroundMusic` has been removed from sfall since 4.3.4.
* `DebugMode=0` because after some long gameplay session, the player may have an unnecessary large debug.log in the game folder. Better to only enable it upon request for actual debugging for issue reports.

### CODE REVIEW

**Comment by burner1024** [comment]
Thank you!

File: release/upu-install.bat
Comment (burner1024) [inline release/upu-install.bat:1]: This one changed line breaks too?

Comment (NovaRain) [reply release/upu-install.bat:1]: It's probably due to I changed `core.autocrlf` git setting on my end. The batch for Windows with CRLF shouldn't have problems on system?

Comment (burner1024) [reply release/upu-install.bat:1]: What system?
I just wondered why it shows all changed. If it's crlf now, I guess it's just as well.

Comment (NovaRain) [reply release/upu-install.bat:1]: I mean running the batch on Windows.

**Pull request merged**